### PR TITLE
Consolidated upload caches directory url construction

### PR DESCRIPTION
**Pull request by alfiehanssen**

### CODE REVIEW

**Comment by alfiehanssen** [comment]
@huebnerob @ghking  this is ready for review

**Comment by alfiehanssen** [comment]
@ghking @huebnerob Hook me up? 
:neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard:

File: VIMUpload/Categories/NSURL+Extensions.m [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+#import "NSURL+Extensions.h"
+
+@implementation NSURL (Extensions)
+
+// TODO: consolidate the below two methods [AH] 9/9/2015
```
Comment (huebnerob) [inline VIMUpload/Categories/NSURL+Extensions.m:31]: why are they even duplicated, is it just the documents or temporary directory choice after `if (groupURL == nil)`?

**Comment by huebnerob** [comment]
:bread:

**Pull request closed without merging**